File: programmers/Level 1/두 정수 사이의 합/solution.js
function solution(a, b) {
    var answer = 0;
    var mid=0;
    
    if(a===b){
        return a;
    }else if(a>b){
        for(var i=b+1; i<a; ++i)
            mid+=i;
    }else if(a<b){
        for(var i=a+1; i<b; ++i)
            mid+=i;
    }
    
    return a+mid+b;
}

File: programmers/Level 1/두 정수 사이의 합/solution.java
class Solution {
  public long solution(int a, int b) {
      long answer = 0;
      long mid = 0;
      
      if(a==b){
          return a;
      }else if(a>b){
          for(int i=b+1; i<a; ++i)
              mid+=i;
          
      }else if(a<b){
          for(int i=a+1; i<b; ++i)
              mid+=i;
      }
      return a+mid+b;
  }
}